### ISSUE TITLE
Not recognizing all packages (nor local ones)

### ISSUE THREAD

**Issue by imLew** [fannheyward/coc-julia#45]
The langserver seems to handle most packages but not all. In particular I'm using `DrWatson` and keep getting a "Missing Reference" warning even though the package is in both the base environment and in the project in which I am working.

Also not sure whether this is an issue with coc-julia or if it's the langserver, but I currently also get missing reference errors for things that are not defined in the same directory (but are defined in the same julia project, git repo and nvim pwd).

**Comment by originalsouth** [comment]
Can confirm the problem on two different setups.

**Comment by fannheyward** [comment]
Should be fixed in v0.6.0 with #67 , please give it a try.

**Comment by imLew** [comment]
When I first tried it `CocInfo` contained an error message saying that 'PackageCompiler' was not found. This was fixed by activating `~/.config/coc/extensions/node_modules/coc-julia/server/compile_env/` in julia and adding it, followed by `Pkg.instantiate()`. PackageCompiler was removed as a dependence in 554fdaaa87c505aea06dec6a1bbc73d074e5a099

This seems to have worked. There are no longer missing reference warnings for packages that are installed in current environment. However functions and modules in the working directory are still not found.
